clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/operations/ChainResolver.java

```diff
@@ -85,14 +85,16 @@ public ChainResolver(final OperationsCodec<K, V> codec) {
    */
   public abstract Map<K, ValueHolder<V>> resolveAll(Chain chain, long now);
 
+  public abstract Map<K, ValueHolder<V>> resolveAll(Chain chain);
```

Comment:
Should this have javadeoc?  At least enough to ensure this method can be differentiated from the one above.